test/runtests.jl

```diff
@@ -5,12 +5,21 @@ using TimeSpans, Dates, Onda, Intervals
 using JSON3, Arrow
 using Aqua
 
+@static if isdefined(Onda, :SignalV2)
```

Comment:
Always `@static` at top-level:
```suggestion
if isdefined(Onda, :SignalV2)
```